Makefile

```diff
@@ -0,0 +1,24 @@
+# This is a make file to help simplify some of the docker commands we use for running things.
+
+build: 
+	docker-compose build
+
+start:
+	docker-compose up -d
+
+start-nobackground:
+	docker-compose up
+
+stop:
+	docker-compose stop
+
+rspec:
+	docker-compose run -e "RAILS_ENV=test" web rspec spec/
+
+db_dev_migrate:
+	docker-compose run web rake db:create && docker-compose run web rake db:migrate
+
+db_dev_drop:
+	docker-compose run web rake db:drop
+
+db_dev_reset: db_dev_drop	db_dev_migrate
```

Comment:
This being a tab character is important. For reasons I don't remember, when a make target references more than one other make target it needs to be separated by a tab.